core/templates/dev/head/pages/admin-page/jobs-tab/admin-jobs-tab.directive.ts

```diff
@@ -20,9 +20,7 @@ require('domain/utilities/UrlInterpolationService.ts');
 
 require('pages/admin-page/admin-page.constants.ts');
 
-var oppia = require('AppInit.ts').module;
-
-oppia.directive('adminJobsTab', [
+angular.module('jobsTabModule').directive('adminJobsTab', [
```

Comment:
adminJobsTabModule

Reply:
I have made some explanation in the [below comment](https://github.com/oppia/oppia/pull/7090#discussion_r300283540)